docs/tutorials.rst

```diff
@@ -36,6 +36,7 @@ The following tutorials (or sections thereof) explain how to use and extend spec
 * :doc:`How do I set up a basic Bayesian optimization routine?<notebooks/expected_improvement>`
 * :ref:`How do I make a custom acquisition function?<notebooks/failure_ego:Create a custom acquisition function>`
 * :ref:`How do I customize the model optimization routine?<notebooks/failure_ego:Create a custom optimize method>`
+* :doc:`How do I recover a failed optimization loop?<notebooks/recovering_from_errors>`
```

Comment:
what's the difference between `:ref:` and `:doc:`?

Reply:
`:ref:` points to a section in a doc, `:doc:` points to a whole file